_std/macros/wire_panel.dm

```diff
@@ -0,0 +1,2 @@
+#define WPANEL_CUSTOM_ACT(C, F, B) new /datum/wirePanel/wireActs(control=C, to_fix=F, to_break=B)
+#define WPANEL_INDICATOR(W, C, A, I) new /datum/wirePanel/indicatorDefintion(control=W, color=C, active=A, inactive=I)
```

Comment:
either give these sane argument names or just factor them out, not sure what they're for other than making it slightly harder to read

Reply:
Guilty of cargo-culting other macros. I'll poke at this and the parent PR to see if I had more reasoning/use out of these, or otherwise just factor out.